src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/PricesSpreadsheetGenerator.kt

```diff
@@ -32,10 +30,14 @@ class PricesSpreadsheetGenerator(
   internal fun generate(supplier: Supplier, startDate: LocalDate): File {
     val dateGenerated = timeSource.date()
 
-    XSSFWorkbook(template.get()).use { workbook ->
+    SXSSFWorkbook().use { workbook ->
```

Comment:
As you can see above we no longer use the template and now start using the streaming version of the Apache POI API.  By default this process 100 rows at a time then writes them to disk instead of just holding them all in memory.